src/web/aws-cloudwatch/CloudWatch.jsx

```diff
@@ -0,0 +1,145 @@
+      {
+        key: 'authorize',
+        title: 'AWS CloudWatch Authorize',
+        component: (<CloudWatchAuthorize onSubmit={this.handleSubmit}
+                                         onChange={this.handleFieldUpdate}
+                                         getValue={this.getFormData} />),
+      },
+      {
+        key: 'kinesis-setup',
+        title: 'AWS CloudWatch Kinesis Setup',
+        component: (<CloudWatchKinesis onSubmit={this.handleSubmit}
+                                       onChange={this.handleFieldUpdate}
+                                       getValue={this.getFormData}
+                                       hasStreams />),
+      },
+      {
+        key: 'health-check',
+        title: 'AWS CloudWatch Health Check',
+        component: (<CloudWatchHealthCheck onSubmit={this.handleSubmit} />),
+      },
+      {
+        key: 'review',
+        title: 'AWS CloudWatch Review',
+        component: (<CloudWatchReview onSubmit={this.handleSubmit} getAllValues={this.getAllFormData} />),
+      },
+    ];
+
+    this.state = {
+      currentStep: 'authorize',
+      enabledSteps: ['authorize'],
+      formData: {},
+      wizardSteps: this.wizardWithDisabledSteps(),
+    };
+
+    this.availableSteps = this.wizardSteps.map(step => step.key);
+  }
+
+  static propTypes = {
+    params: PropTypes.shape({
+      step: PropTypes.string,
+    }),
+  }
+
+  wizardWithDisabledSteps = () => {
+    return this.wizardSteps.map((step) => (
+       {
+        ...step,
+        disabled: this.isDisabledStep(step.key),
+      }
+    ));
+  }
+
+  getFormData = (value) => {
+    return this.state.formData[value];
+  }
+
+  getAllFormData = () => this.state.formData;
+
+  isDisabledStep = (step) => {
+    if (!this.state) {
+      return true;
+    }
+
+    const { enabledSteps } = this.state;
+
+    if (!enabledSteps || enabledSteps.length === 0) {
+      return true;
+    };
+
+    return !enabledSteps.includes(step)
+  };
+
+  handleFieldUpdate = ({ target }) => {
+    const { formData } = this.state;
+
+    this.setState({
+      formData: {
+        ...formData,
+        [target.id]: target.value
+      }
+    })
+  }
+
+  handleSubmit = (formData) => {
+    formData.preventDefault();
+
+    const { currentStep, enabledSteps } = this.state;
+    const nextStep = this.availableSteps.indexOf(currentStep) + 1;
+
+    if (this.wizardSteps[nextStep]) {
+      const key = this.wizardSteps[nextStep].key;
+
+      this.setState({
+        enabledSteps: [...enabledSteps, key],
+        currentStep: key,
+      }, () => {
+        this.setState({
+          wizardSteps: this.wizardWithDisabledSteps(this.wizardSteps),
+        });
+      });
+    }
+
+    // UserNotification.success(`Form Data Was Submitted`, 'Hurray!');
+  }
+
+  handleStepChange = (currentStep) => {
+    this.setState({
+      currentStep,
+    })
+  }
+
+  render() {
+    const { currentStep, wizardSteps } = this.state;
+
+    return (
+      <Row>
```

Comment:
After a `Row` a `Col` component should follow to ensure the margins of the page are set as they should:

With `Col`:
![Screenshot 2019-06-13 at 14 16 51](https://user-images.githubusercontent.com/716185/59431761-f4775800-8de5-11e9-8fcd-5b93bae7469e.png)

Without `Col`:
![Screenshot 2019-06-13 at 14 17 09](https://user-images.githubusercontent.com/716185/59431767-fe00c000-8de5-11e9-9b28-1b7a786cb94c.png)